decidim-templates/app/commands/decidim/templates/admin/create_proposal_answer_template.rb

```diff
@@ -38,8 +38,8 @@ def identify_templateable_resource
           when "organizations"
             @form.current_organization
           when "components"
-            component = Decidim::Component.where(manifest_name: :proposals).find(resource.last)
-            component.participatory_space.decidim_organization_id == @form.current_organization.id ? component : nil
+            component = Decidim::Component.find_by(id: resource.last)
```

Comment:
The main idea here was to make sure the requested component is a proposal.
by the way, the find_by(id: ... ) ... is much easier written: find(...)